docs/zh/latest/FAQ.md

```diff
@@ -21,73 +21,97 @@ title: 常见问题
 #
 -->
 
-## 为什么要做 API 网关？不是已经有其他的开源网关了吗？
+## 为什么我们需要一个新的 API 网关？不是已经有其他的开源网关了吗？
 
-微服务领域对 API 网关有新的需求：更高的灵活性、更高的性能要求，以及云原生的贴合。
+随着企业向云本地微服务的发展，对高性能、灵活、安全、可扩展的API网关的需求越来越大，一些之前的网关已经逐渐无法满足当今企业的需求。
```

Comment:
```suggestion
随着企业向云本地微服务的发展，对高性能、灵活、安全、可扩展的 API 网关的需求越来越大，一些之前的网关已经逐渐无法满足当今企业的需求。
```
We need add black space between English and Chinese, you could refer to https://github.com/sparanoid/chinese-copywriting-guidelines

Reply:
> 随着企业向云原生微服务的发展，需要有一个高性能、灵活、安全和可扩展的 API 网关。

![image](https://user-images.githubusercontent.com/99874763/160737152-c14d754e-2906-4aea-9a41-a993546d3aad.png)
I think the answer to this question would be better with APISIX versus other gateways.